pkg/daemon/ceph/agent/flexvolume/controller_test.go

```diff
@@ -724,6 +724,7 @@ func TestGetAttachInfoFromMountDir(t *testing.T) {
 						StorageClassKey: "storageClass1",
 						PoolKey:         "pool123",
 						ImageKey:        "pvc-123",
+						DataPoolKey:     "",
```

Comment:
if this setting is missing everything will still work, right? This will make backward compatibility simpler

Reply:
Yes, if it is empty no `--data-pool=POOL` will be added to the `rbd create` command.